tracer.go

```diff
@@ -23,4 +23,21 @@ type Tracer interface {
 	//         SetTag("lucky_number", 42)
 	//
 	StartTrace(operationName string) Span
+
+	// ImplementationID returns information about the OpenTracing
+	// implementation backing the Tracer. The return value should not change
+	// over the life of a particular Tracer instance.
+	ImplementationID() *ImplementationID
+}
+
+// ImplementationID is a simple, extensible struct that describes an
+// OpenTracing implementation.
+type ImplementationID struct {
+	// The (stable) name of the implementation. E.g., "Dapper" or "Zipkin". The
+	// Name should not reflect the host language or platform.
+	Name string
+
+	// Version may take any form, but SemVer (http://semver.org/) is strongly
+	// encouraged.
+	Version string
```

Comment:
I agree with @bcronin, it's the OpenTracing version that the original ticket was asking for. We said it would be best done via Capabilities API rather than a version. At the moment, however, there doesn't seem to be any aspects of the API that are optional and require introspection by the client code.